### PR TITLE
Implement mongodb connect

### PR DESCRIPTION
**Pull request by shabanzadeh**

### CODE REVIEW

File: .dockerignore
Comment (syedaffanhamdani) [inline .dockerignore:1]: +1

File: Dockerfile
Comment (syedaffanhamdani) [inline Dockerfile:1]: +1

File: README.md [hunk 1]
```diff
@@ -19,11 +20,12 @@ To set up a virtual environment for this application, follow these steps:
 
 on Windows:
 
-    `cd venv/Scripts/activate`
+` cd venv/Scripts/activate`
 
 on macOS/Linux
 
-    `cd venv/bin/activate`
+`cd venv/bin/activate`
```
Comment (syedaffanhamdani) [inline README.md]: I think it should be source venv/bin/activate

File: README.md [hunk 2]
```diff
@@ -35,6 +37,24 @@ on macOS/Linux
 
     `pip install fastapi mongodb uvicorn`
```
Comment (syedaffanhamdani) [inline README.md]: unsure about step 2, all dependencies should be in requirements file and step1 should be enough

File: README.md [hunk 3]
```diff
@@ -35,6 +37,24 @@ on macOS/Linux
...
 
+
+## docker-compose between FastAPI and MongoDB
+
+Create docker-compose.yml file in the folder from which you launch your container.
```
Comment (syedaffanhamdani) [inline README.md]: this line can be removed. File is part of repo

File: docker-compose.yml
Comment (syedaffanhamdani) [inline docker-compose.yml:1]: +1

**Review by syedaffanhamdani — COMMENTED** [review]
Very nice work overall. I could not find tests for the endpoints.

File: README.md (hunk 3 above)
Comment (shabanzadeh) [reply README.md]: I have done it

File: README.md (hunk 1 above)
Comment (shabanzadeh) [reply README.md]: I have done it

File: README.md (hunk 2 above)
Comment (shabanzadeh) [reply README.md]: I deleted it

**Pull request merged**